docs/products/flink/howto/manage-flink-tables.rst

```diff
@@ -0,0 +1,74 @@
+Manage tables in Apache Flink® applications
+===========================================
+
+Aiven for Apache Flink® allows you to map source and target data structures as `Flink tables <https://nightlies.apache.org/flink/flink-docs-stable/docs/dev/table/sql/create/#create-table>`_ and use transformation statements to transform data. Some of the operations you can perform on a table include:
+
+* Import existing tables
+* Add new tables
+* Clone tables
+* Edit tables
+* Delete tables
+
+Before performing any operation on a table in a Flink application, you must stop the application. To stop an application, navigate to the **Applications** tab on your Aiven for Apache Flink® service, select the desired application from the list, and click **Stop Deployment**.
+
+Add new table
+--------------
+
+Follow these steps add a new table to an application: 
+
+1. Open the application to which you want to add a new table.
```

Comment:
I know it's coming, but it would be good to reference the "application" concept document when we mention the app